pkg/apiserver/api_installer.go

```diff
@@ -62,6 +62,21 @@ type documentable interface {
 	SwaggerDoc() map[string]string
 }
 
+// toDiscoveryKubeVerb maps an action.Verb to the logical kube verb, used for discovery
+var toDiscoveryKubeVerb = map[string]string{
+	"CONNECT":          "", // do not list in discovery. TODO: is this correct?
+	"DELETE":           "delete",
+	"DELETECOLLECTION": "deletecollection",
+	"GET":              "get",
+	"LIST":             "list",
+	"PATCH":            "patch",
+	"POST":             "create",
+	"PROXY":            "proxy",
+	"PUT":              "update",
+	"WATCH":            "watch",
+	"WATCHLIST":        "watch",
```

Comment:
Compare https://github.com/kubernetes/kubernetes/blob/3454a8d52cc684dcbae9b4a924105fd3e869a8d6/pkg/controller/namespace/namespace_controller_utils.go#L301-L301. In the namespace controller we try `deletecollection` first and fall back to single `delete`s. @liggitt @deads2k has anything changed that this is not necessary anymore?